docker-images/ziti-console-assets/Dockerfile

```diff
@@ -11,12 +11,17 @@ WORKDIR /usr/src/app
 RUN npm install -g @angular/cli@16.0.0-next.0
 
 COPY . .
-RUN npm install --omit=optional
-RUN ng build ziti-console-lib
-RUN ng build ziti-console --base-href ${BASE_HREF} --configuration "production"
+
```

Comment:
this doesn't change the image contents, but accelerates builds by mounting a BuildKit cache